1/contrib/openshift/configuration/jobs/OpenShift Sample/config.xml

```diff
@@ -13,66 +13,24 @@
   <builders>
 
     <com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <depCfg>frontend</depCfg>
-      <namespace>test</namespace>
-      <replicaCount>0</replicaCount>
-      <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
 
-    <com.openshift.jenkins.plugins.pipeline.OpenShiftDeploymentVerifier>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <depCfg>frontend</depCfg>
-      <namespace>test</namespace>
-      <replicaCount>0</replicaCount>
-      <authToken></authToken>
-    </com.openshift.jenkins.plugins.pipeline.OpenShiftDeploymentVerifier>
-
     <com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <bldCfg>frontend</bldCfg>
-      <namespace>test</namespace>
-      <authToken></authToken>
-      <followLog>true</followLog>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftBuilder>
 
-    <com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <depCfg>frontend</depCfg>
-      <namespace>test</namespace>
-      <replicaCount>1</replicaCount>
-      <authToken></authToken>
-    </com.openshift.jenkins.plugins.pipeline.OpenShiftScaler>
-    
     <com.openshift.jenkins.plugins.pipeline.OpenShiftDeploymentVerifier>
-      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
-      <depCfg>frontend</depCfg>
-      <namespace>test</namespace>
-      <replicaCount>1</replicaCount>
-      <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftDeploymentVerifier>
```

Comment:
assuming we set the checkbox on the build step, we don't need this verification step.